archivessnake/unpublish_processing_note.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import os
+import json
+import time
+from asnake.aspace import ASpace
+
+
+aspace = ASpace(
+              baseurl='',
+              user='',
+              password=''
+    )
+
+repo = aspace.repositories(2)
+start_time = time.time()
+
+
+def get_resources_notes():
+    for object_type in ['resources']:
+        for object in getattr(aspace, object_type):
+            if (object.jsonmodel_type == 'resource' and object.id_0.startswith(('FA'))):
+                unpublish_notes(object)
```

Comment:
Can't you do this more simply?
```suggestion
    for resource in aspace.resources:
        if resource.id_0.startswith(('FA'))):
            unpublish_notes(resource)
```

If you were looking to do both resources and AO's in the same loop method, you could do something like the above, but I actually think that's more confusing than just having two separate (albeit similar) functions.

Reply:
I simplified the resource function, but I left the two functions separate.